eng/pipelines/templates/stages/archetype-cpp-release.yml

```diff
@@ -100,6 +100,24 @@ stages:
                           Write-Host "##vso[task.setvariable variable=PrBranchName]$branchName"
                         displayName: Set fork branch name
 
+                      - pwsh: |
+                          $packageSpec = Get-Content -Raw -Path "$(Pipeline.Workspace)/packages/${{artifact.Name}}/package-info.json" | ConvertFrom-Json
+                          $version = $packageSpec.version
+                          $title = "[${{ artifact.VcpkgPortName }}] publish version $version"
+
+                          if ('$(VcpkgPRTitle)') { 
+                            Write-Host "Using queue time PR title"
```

Comment:
Output the `VcpkgPRTitle` string to the console as part of this message.

Reply:
Edit: Nevermind, looks like after the `if-block` below, we output `$title` already.